packages/imperative/CHANGELOG.md

```diff
@@ -2,11 +2,15 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Resolved an issue where extraneous base profiles were being added to project configs if a nested profile property was updated. [#2400](https://github.com/zowe/zowe-cli/pull/2400)
```

Comment:
- Please spell out "configs".
- If accurate/appropriate, I'd change "if" to "when": " ... **when** a nested profile ..."